src/_help/hubs/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
+
+With hubs, your API ecosystem is displayed on a single page, where documentation can be sorted by categories.
+
+Hubs also provide a unified changelog, making it easy to identify all changes across all the documentation it contains at a glance. Search is also global, allowing you to search across all documentation.
+
+## Create a hub
+
+In your dashboard, click on Hubs then on Create a hub.
+
+![](/images/help/create-hub.png)
+
+You will be asked to choose a name for the hub, which will be visible to everyone. The slug is the portion of the URL that you can customize. You can also choose to use a custom domain instead. Finally, you can decide whether this hub is visible to everyone or only to members of your organization.
```

Comment:
I'm not sure about the mention on custom domains. IIUC, the slug will be valid no matter the domain, whether custom or on bump.sh? Also, the custom domain settings do not happen here, and I feel the `You can also choose to use a custom domain instead.` could be misleading.

Reply:
The custom domain part is important here. @ChristopheDujarric no the Hub slug is not visible if you use a custom domain on your hub.

Maybe we can mention `bump.sh` domain in the sentence talking about the slug:
- On bump.sh, your hub will be available on this URL `https://bump.sh/hub/<your-own-slug>`
- With a custom domain, your hub will be available on `https://www.custom.domain.com`

Reply:
👍